intel-sgx/sgxs/src/sgxs.rs

```diff
@@ -418,8 +418,8 @@ impl<W: Write> SgxsWrite for W {
         let mut buf = [0u8; 64];
         unsafe {
             let (tag, headerdst) = buf.split_at_mut(8);
-            let tag = &mut *(&mut tag[0] as *mut _ as *mut u64);
-            let headerdst = &mut headerdst[0] as *mut _;
+            let tag = &mut *(&mut tag[0..8] as *mut _ as *mut u64);
```

Comment:
Wow, so new nightly compiler will warn this ?

Reply:
Yes it surprised me too!